src/Pos/Communication/Limits.hs

```diff
@@ -283,6 +284,12 @@ instance MessageLimited (MsgHeaders ssc) where
         return $
             MsgHeaders <$> vectorOf Const.recoveryHeadersMessage headerLimit
 
+-- TODO: Update once we move to CBOR.
+instance MessageLimitedPure MsgSubscribe where
+    msgLenLimit = 0
```

Comment:
But doesn't Void have a non-0 encoding in CBOR? Or is Void a special case in the comms where it will not send anything?